packages/core/src/scss/components/foundation/forms/_forms.scss

```diff
@@ -216,7 +216,6 @@
 
     .loading-skeleton {
         animation: grows 150ms ease-out;
-        overflow: hidden;
```

Comment:
The actual content after loading has a slideDown effect, which is why we have `will-change: max-height;` here.
I can see why the CVV tooltip is affected by this line. However, removing it may cause the animations we intended to break.
I'll put this to the test and see if there's another way and get back to you tomorrow.

Reply:
@bc-peng Thank you

Reply:
Thanks Peng 🙇

Reply:
I tested and found out we could move `overflow:hidden` to the keyframes.
Could you please keep this line removed and edit the keyframes on L222 as below:
```
@keyframes grows {
        0% { max-height: $floating-label-input-height * 4; overflow: hidden; }
        100% { max-height: 2000px; overflow: visible; }
    }
```
Testing video:

https://user-images.githubusercontent.com/88361607/209022851-2fe7bf1f-4a1a-4c91-b746-9f254cf7234c.mov

Reply:
Thank you for your investigation. Fixed